packages/portfolio-contract/test/snapshots/portfolio.flows.test.ts.md

```diff
@@ -1231,3 +1231,170 @@ Generated by [AVA](https://avajs.dev).
+          value: 300n,
+        },
+        opts: undefined,
+      },
+      {
+        _cap: 'agoric11014',
+        _method: 'transfer',
+        address: {
+          chainId: 'axelar-6',
+          encoding: 'bech32',
+          value: 'axelar1dv4u5k73pzqrxlzujxg3qp8kvc3pje7jtdvu72npnt5zhq05ejcsn5qme5',
+        },
+        amount: {
+          denom: 'ubld',
+          value: 50n,
+        },
+        opts: {
+          memo: '{"destination_chain":"arbitrum","destination_address":"0xef8651dD30cF990A1e831224f2E0996023163A81","payload":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,32,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"type":1,"fee":{"amount":"50","recipient":"axelar1zl3rxpp70lmte2xr6c4lgske2fyuj3hupcsvcd"}}',
+        },
+      },
+      {
+        _cap: 'noble11028',
+        _method: 'depositForBurn',
+        denomAmount: {
+          denom: 'uusdc',
+          value: 300n,
+        },
+        destinationAddress: 'eip155:42161:0x126cf3AC9ea12794Ff50f56727C7C66E26D9C092',
+      },
+      {
+        _cap: 'agoric11014',
+        _method: 'transfer',
+        address: {
+          chainId: 'axelar-6',
+          encoding: 'bech32',
+          value: 'axelar1dv4u5k73pzqrxlzujxg3qp8kvc3pje7jtdvu72npnt5zhq05ejcsn5qme5',
+        },
+        amount: {
+          denom: 'ubld',
+          value: 100n,
+        },
+        opts: {
+          memo: '{"destination_chain":"arbitrum","destination_address":"0x126cf3AC9ea12794Ff50f56727C7C66E26D9C092","payload":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,32,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,2,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,64,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,202,199,255,168,44,15,67,235,176,252,17,252,211,33,35,236,164,102,38,207,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,64,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,68,9,94,167,179,0,0,0,0,0,0,0,0,0,0,0,0,135,135,11,202,63,63,214,51,92,63,76,232,57,45,105,53,11,79,164,226,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,44,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,135,135,11,202,63,63,214,51,92,63,76,232,57,45,105,53,11,79,164,226,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,64,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,132,97,123,160,55,0,0,0,0,0,0,0,0,0,0,0,0,202,199,255,168,44,15,67,235,176,252,17,252,211,33,35,236,164,102,38,207,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,44,0,0,0,0,0,0,0,0,0,0,0,0,18,108,243,172,158,161,39,148,255,80,245,103,39,199,198,110,38,217,192,146,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"type":1,"fee":{"amount":"100","recipient":"axelar1zl3rxpp70lmte2xr6c4lgske2fyuj3hupcsvcd"}}',
```

Comment:
the payload isn't really readable here.

What do you think about an additional test just for the payload?
Or for the `calls`?